creator-node/src/config.js

```diff
@@ -262,7 +262,7 @@ const config = convict({
     doc: 'If we should print logs from sequelize',
     format: Boolean,
     env: 'printSequelizeLogs',
-    default: true
+    default: false
```

Comment:
should we be defaulting to true? seems like we'll be missing some logs if this is set to false

Reply:
@SidSethi @vicky-g this and the commonEnv.sh change is necessary so this doesn't send sequelize debug sql queries to loggly. with kube and fluentd this was never sent over. it was primarily a debugging tool and commonEnv.sh now enables it for local development but turns it off for live stage/prod environments

Reply:
gotcha, thanks for clarifying